content/using.html

```diff
@@ -66,25 +66,28 @@
 
   {% endcall %}
 
-  {% call subsection('Channels')%}
+  {% call subsection('Labels')%}
 
-  Each file within a package may be tagged in one or more channels. A file may also not be tagged in any channels.
-  One common use of channels is to divide a package's files into development, testing, and main channels called 
-  dev, test, and main. This allows package authors to upload files for development or testing purposes without
-  affecting non-development users.
-
-  At [https://**pypi**.anaconda.org/conda](https://pypi.anaconda.org/conda) a channel
-  dropdown lists all the available channels in this user namespace.
+  Each file within a package may be tagged with one or more labels, or not tagged at all. 
+  Labels are commonly used to divide a package's files into sections labeled development, 
+  testing (and/or other custom labels), and main or dev, test, main or other custom label. 
```

Comment:
Perhaps we could say:

Each file within a package may be tagged with one or more labels, or not tagged at all. 
A package's files can be sorted into categories such as development, testing, and main with labels such as "dev" "test" and "main". Then developers can set their machines to install all updates, testers can set their machines to install only "test" and "main" updates, and other users can set their machines to install only "main" updates. Any other custom labels can be used instead so that the label system can fit any workflow.